### PR TITLE
Updated Community members; other minor changes

### LINKED ISSUE #227: [Members] Update position holders of communities
### Description
<write your description> Since the community co-ordinators and heads of each community have been updated, it is necessary to reflect that in the website.

### Assets and elaboration

To be updated soon.

### Concerning files/folders

https://github.com/houseofgeeks/hg/tree/front-end/client/src/components/Members/Community

### PR DESCRIPTION
**Pull request by SrijanSriv**
### Fixes #227
<Do not delete this template>
Description: The latest batch of Community members have been added to the respective places. Furthermore, I realized not all the images were being rendered since some folders had spaces in their paths (e.g. ./images/members/ webd and appd/...), so after some discussion, that was fixed as well. Attached is the same issue.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes that were demanded in the linked issue.
- [x] I've tested my code on a Chromium-based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
image

### CODE REVIEW

File: client/src/components/Members/Community/ElcMembersList.js [hunk 1]
```diff
@@ -2,19 +2,20 @@ export const ElcMembersLists = [
...
-            mail: 'devansh.btech.ec18@iiitranchi.ac.in',
-            linkedin: 'https://www.linkedin.com/in/devansh-srivastava-958766167'
+            mail: 'shaswat.btech.ec19@iiitranchi.ac.in',
+            linkedin: 'https://www.linkedin.com/in/shashwat-pratap-470b7116b',
+            github: 'https://github.com/shashwat0426'
       }
+      // ,{
+      //       id: 1,
+      //       img: './images/Members/Core Electronics/devansh.jfif',
+      //       name: 'Devansh Srivastava',
```
Comment (EmperorYP7) [inline client/src/components/Members/Community/ElcMembersList.js]: Devansh Srivastava is not holding any posts in House of Geeks anymore. I think you should remove this segment.

File: client/src/components/Members/Community/Electronics.members.js [hunk 2]
```diff
@@ -16,6 +17,10 @@ function ElectronicsMembers() {
                         <a className="text-white" target="_blank" rel="noopener noreferrer" href={elcList.linkedin}>
                               <LinkedIn className="icons-members"></LinkedIn>
 
+                        </a>
+                        <a className="text-white" target="_blank" rel="noopener noreferrer" href={elcList.github}>
```
Comment (EmperorYP7) [inline client/src/components/Members/Community/Electronics.members.js:21]: Good job! :smile:

File: client/src/components/Members/Community/WebMembersList.js [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
-            position: 'Community Head',
-            mail: 'vpankaj1998official@gmail.com',
-            github: '',
-            linkedin: 'linkedin.com/in/iampankajvaghela'
-      },
+export const WebMembersLists = [
+      // {
+      //       id: 0,
+      //       img: './images/Members/WebandAppDev/pankaj.jfif',
+      //       name: 'Pankaj Waghela',
```
Comment (EmperorYP7) [inline client/src/components/Members/Community/WebMembersList.js]: Pankaj Waghela is also not holding any position. Might remove this as well.

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
LGTM. Just a bit of cleanup required.

File: client/src/components/Members/Community/WebMembersList.js (hunk 3 above)
Comment (SrijanSriv) [reply client/src/components/Members/Community/WebMembersList.js]: Ohh about that, yeah the comments were redundant but i thought maybe not get rid of the photos cuz maybe we could create like a wall of ex-members in the future. Let me see this anyway.

**Comment by SrijanSriv** [comment]
Wait I've never had a review update, do I create a new pr? :sweat_smile:

**Comment by EmperorYP7** [comment]
> Wait I've never had a review update, do I create a new pr? 😅

Nope. Just push new commits to `SrijanSriv:front-end`

File: client/src/components/Members/Community/WebMembersList.js (hunk 3 above)
Comment (EmperorYP7) [reply client/src/components/Members/Community/WebMembersList.js]: Oh yea, remove the photos as well if you will. It'd marginally reduce the deployment size.

**Comment by SrijanSriv** [comment]
Right. Good to know that Github takes the stress here.

**Review by EmperorYP7 — APPROVED** [review]
LGTM

**Comment by EmperorYP7** [comment]
Good job @SrijanSriv! Welcome to the contributor's club 🌟

**Comment by EmperorYP7** [comment]
Good job @SrijanSriv! Welcome to the contributor's club 🌟

@all-contributors please add @SrijanSriv for code

**Comment by EmperorYP7** [comment]
The website isn't updated. https://houseofgeeks.netlify.app/members

I think @ankiiitraj needs to retrigger the deployment.

**Comment by tusharjain0022** [comment]
@all-contributors please add @SrijanSriv for code

**Pull request merged**